### PR TITLE
Kristina Wald's Intro

### PR DESCRIPTION
**Pull request by kristinawald**
I had some initial trouble setting up the project in R, but that was resolved by getting the git installation into the right place.  I also had to get used to the process of committing, including saving the file, committing, and pushing, but once I did this a few times it started to come pretty intuitively.  Otherwise, the main difficulty I had was including the picture; I had to try several times to get this to work properly, and I found the instructions for that part rather confusing/incomplete (for example, it wasn't obvious to me what the URL should be or what Alt Text means).  However, I was able to get it to work after some help from the TA.  Other than that, incorporating the rest of the elements worked pretty easily!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+* Watch _**The Bachelor**_
+* Sleep
+
+# Fun Facts
+1. I go to Starbucks every day, but **don't really drink coffee!**
+1. Visiting Antarctica is on my bucket list.
+1. I lived in New York for 6 years and _**fell madly in love with it!**_
+
+# LinkedIn Page
+[Click here](https://www.linkedin.com/in/kristina-wald-77a4a763/)
```
Comment (bensoltoff) [inline README.md:28]: Nice addition of a URL!

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+# Age
+25
+
+# Where I'm from
+Hyde Park - *grew up practically right on campus!*
+
+# What I'm studying
+Behavioral Science at Booth
+
+# Research interests
```
Comment (yihuang97) [inline README.md:12]: These are also the topics that I am interested in!

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+* Person perception
+* Impression formation
+* Social interaction
+
+# Favorite things to do
+* Take walks *(especially along Lake Michigan!)*
+* Watch _**The Bachelor**_
+* Sleep
+
+# Fun Facts
```
Comment (yihuang97) [inline README.md:22]: One of my friends also likes to go to Starbucks and she only drinks non caffeinated frappuccino

File: README.md [hunk 4]
```diff
@@ -1 +1,34 @@
...
+1. I go to Starbucks every day, but **don't really drink coffee!**
+1. Visiting Antarctica is on my bucket list.
+1. I lived in New York for 6 years and _**fell madly in love with it!**_
+
+# LinkedIn Page
+[Click here](https://www.linkedin.com/in/kristina-wald-77a4a763/)
+
+# Picture of Me
+![Me](Wald-2644.jpg)
+
```
Comment (yihuang97) [inline README.md:32]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |       X    |              |            |

Overall it's a very great post and I'm glad to know more about you!

File: README.md [hunk 5]
```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+Hey there!  My name is Kristina Wald.  Here are some quick facts about me:
+
+# Age
+25
+
+# Where I'm from
```
Comment (bensoltoff) [inline README.md:6]: Good practice is to leave a line break between headers and blocks of text (or really between two different types of markdown content). Avoids formatting issues later on

File: README.md [hunk 6]
```diff
@@ -1 +1,34 @@
...
+Behavioral Science at Booth
+
+# Research interests
+* Person perception
+* Impression formation
+* Social interaction
+
+# Favorite things to do
+* Take walks *(especially along Lake Michigan!)*
+* Watch _**The Bachelor**_
```
Comment (bensoltoff) [inline README.md:19]: I'm kind of annoyed with Peter as the next bachelor. I like him, his mom seems just the right amount of crazy, but Mike would have injected some variety and fun.

Or a Property Brother. That would also be intriguing.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |     x      |              |            |

* Nice variety of markdown elements
* Workflow is established. Next step - write R code!

**Pull request closed without merging**